### PR TITLE
Port to jbuilder build tool

### PR DESCRIPTION
**Pull request by lindig**
This commit replaces the Oasis for Jbuilder as the build tool. This
required a minimal code change for the tests.

Signed-off-by: Christian Lindig <christian.lindig@citrix.com>

### CODE REVIEW

**Comment by kc284** [comment]
I think you need to update the travis file as well. The easiest workaround is to copy ocaml-qmp.opam to opam under `script:`.

File: README.md [hunk 1]
```diff
@@ -1,21 +1,33 @@
-ocaml-qmp
-=========
+
+# ocaml-qmp
```
Comment (mseri) [inline README.md]: Oh, sorry, just noticed. Can you use `qmp` here?

**Comment by lindig** [comment]
We wil have to remember to update the Opam file in `xs-opam` when we integrate this change.

File: .travis.yml [hunk 2]
```diff
@@ -3,8 +3,12 @@ sudo: required
 install:
   - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
 script:
+  - cp qmp.opam opam
```
Comment (thomassa) [inline .travis.yml]: Why do we need two differently-named copies of this file?
Would a symlink work?

Comment (lindig) [reply .travis.yml]: This is just in the Travis machine and just to make the scripts happy. I don't think this is worth sweating about.

**Pull request merged**